src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -1427,21 +1427,28 @@ private void LoadHomePage()
         {
             if (homePage == null && (startPage != null))
             {
-                homePage = new UI.Views.HomePage();
-                homePage.DataContext = startPage;
-
-                var visibilityBinding = new System.Windows.Data.Binding
+                try
                 {
-                    RelativeSource = new RelativeSource(RelativeSourceMode.FindAncestor, typeof(DynamoView), 1),
-                    Path = new PropertyPath("DataContext.ShowStartPage"),
-                    Mode = BindingMode.OneWay,
-                    Converter = new BooleanToVisibilityConverter(),
-                    UpdateSourceTrigger = UpdateSourceTrigger.Explicit
-                };
+                    homePage = new UI.Views.HomePage();
+                    homePage.DataContext = startPage;
+
+                    var visibilityBinding = new System.Windows.Data.Binding
+                    {
+                        RelativeSource = new RelativeSource(RelativeSourceMode.FindAncestor, typeof(DynamoView), 1),
+                        Path = new PropertyPath("DataContext.ShowStartPage"),
+                        Mode = BindingMode.OneWay,
+                        Converter = new BooleanToVisibilityConverter(),
+                        UpdateSourceTrigger = UpdateSourceTrigger.Explicit
+                    };
 
-                BindingOperations.SetBinding(homePage, UIElement.VisibilityProperty, visibilityBinding);
+                    BindingOperations.SetBinding(homePage, UIElement.VisibilityProperty, visibilityBinding);
 
-                this.newHomePageContainer.Children.Add(homePage);
+                    this.newHomePageContainer.Children.Add(homePage);
+                }
+                catch (Exception ex)
+                {
+                    Log(ex.Message);
+                }
```

Comment:
any idea what exactly is crashing?

Reply:
Agree with @pinzart90 we should know why/what is crashing.
I have seen several times that when opening the latest Dynamo (3.4 or 3.3) the HomePage is NOT being shown and instead the old one is shown but I didn't know exactly why (although is not always happening).

Reply:
@RobertGlobant20 @pinzart90 I have removed the feature flag checking code for DynamoHome in 3.4 release, in that case, Dynamo home would always be the default. In the case it was not initialized correctly, then we display the legacy home. Adding such exception logging would help us understand why it was not working